modules/azurite/testcontainers/azurite/__init__.py

```diff
@@ -33,10 +33,9 @@ class AzuriteContainer(DockerContainer):
             >>> from testcontainers.azurite import AzuriteContainer
             >>> from azure.storage.blob import BlobServiceClient
 
-            >>> with AzuriteContainer() as azurite_container:
-            ...   connection_string = azurite_container.get_connection_string()
+            >>> with AzuriteContainer("mcr.microsoft.com/azure-storage/azurite:3.29.0") as azurite_container:
             ...   client = BlobServiceClient.from_connection_string(
-            ...        connection_string,
+            ...        azurite_container.get_connection_string(),
```

Comment:
For `ArangoDbContainer` is get_conenction_url`, here it's '*_string'. Should we unify that?